modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -47,3 +46,33 @@ reviews-v3-6dfd49b55b-vcwpf       2/2     Running   0          12m
 echo "http://$GATEWAY_URL/productpage"
 ----
 . Copy and paste the output in a web browser to verify the Bookinfo product page is deployed.
+
+.Procedure from Kiali web console
+
+. Obtain the address for the Kiali web console.
+
+.. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.. Navigate to *Networking* -> *Routes*.
+
+.. On the *Routes* page, select the control plane project, for example `istio-system`, from the *Namespace* menu.
++
+The *Location* column displays the linked address for each route.
++
+
+.. Click the link in the *Location* column for Kiali.
+
+.. Click *Log In With OpenShift*. The Kiali *Overview* screen presents tiles for each project namespace.
+
+. In Kiali, click *Graph*.
+
+. Select bookinfo from the *Namespace* list, then select App graph.
++
+image::ossm-kiali-graph-bookinfo.png[Kiali displaying bookinfo application]
++
+If no graph appears in the console, then the application might not have obtained data. To update the data:
++
+* If the empty graph message appears, then click *Display idle nodes*
+* If the no namespace is selected message appears, then change the Traffic Metric Refresh Rate and the Refresh interval.
```

Comment:
```suggestion
* Increasing the time period via the *Duration* menu can help ensure older traffic is captured.  The *Refresh Rate* menu can also be used to refresh more or less often, or not at all.
```

Reply:
As written this is not valid.  If the "no namespace is selected message appears" it means exactly that.  It means no namespace is selected, but we covered that in the step abovethat tells them to select the bookinfo namespace.  Changing the duration or refresh rate would have no effect.

Reply:
Thanks for the review comments, @jshaughn. I've updated the topic, and squashed. The latest edits should now be available in the preview build:

https://deploy-preview-44411--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh.html#ossm-tutorial-bookinfo-verify-install_ossm-create-mesh